crowdin_api/api_resources/dictionaries/resource.py

```diff
@@ -13,7 +13,7 @@ class DictionariesResource(BaseResource):
     Use API to get the list of organization dictionaries and to edit a specific dictionary.
 
     Link to documentation:
-    https://support.crowdin.com/api/v2/#tag/Dictionaries
+    https://develop.crowdin.com/api/v2/#tag/Dictionaries
```

Comment:
These links are broken. The develope**r**.crowdin.com is correct

Reply:
updated